webxrlayers-1.bs

```diff
@@ -907,10 +910,12 @@ To <dfn>determine the layout attribute</dfn> using an {{XRTextureType}} |texture
   1. If |textureType| is {{"texture-array"}} and not all of the session’s [=view|views=] in the [=list of views=] have the same [=recommended WebGL texture resolution=], throw an {{NotSupportedError}} and abort these steps.
   1. If |layout| is {{XRLayerLayout/"mono"}}, return |layout| and abort these steps.
   1. If |layout| is {{XRLayerLayout/"default"}}, run the following steps:
+    1. If the size of [=list of views=] is <code>1</code>, return {{XRLayerLayout/"mono"}} and abort these steps.
     1. If |textureType| is {{"texture-array"}}, return |layout| and abort these steps.
+  1. If |layout| is {{XRLayerLayout/"default"}} or {{XRLayerLayout/"stereo"}}, run the following steps:
     1. If the user agent prefers {{XRLayerLayout/"stereo-left-right"}} layout, return {{XRLayerLayout/"stereo-left-right"}} and abort these steps.
     1. If the user agent prefers {{XRLayerLayout/"stereo-top-bottom"}} layout, return {{XRLayerLayout/"stereo-top-bottom"}} and abort these steps.
-  1. If |textureType| is {{"texture-array"}}, throw {{TypeError}} and abort these steps.
+  1. If |textureType| is {{"texture-array"}} and |layout| is {{XRLayerLayout/"stereo-left-right"}} or {{XRLayerLayout/"stereo-top-bottom"}}, throw {{TypeError}} and abort these steps.
```

Comment:
I don't particularly oppose throwing in this situation, but wanted to point out that a possibly exception is returning a texture array with a single layer. Sounds silly, but based on the call today we may find ourselves doing that in some scenarios anyway? Don't block this PR on it, though.

Reply:
I agree. We don't limit mono layers to texture_2d either. 
I will make the change.